src/PostgREST/DbStructure.hs

```diff
@@ -553,101 +602,9 @@ allColumns tabs =
         info.data_type AS col_type,
         info.character_maximum_length AS max_len,
         info.column_default AS default_value,
-        array_to_string(enum_info.vals, ',') AS enum,
+        coalesce(enum_info.vals, '{}') AS enum,
         info.position
-    FROM (
-        -- CTE based on pg_catalog to get PRIMARY/FOREIGN key and UNIQUE columns outside api schema
-        WITH key_columns AS (
-             SELECT
-               r.oid AS r_oid,
-               c.oid AS c_oid,
-               n.nspname,
-               c.relname,
-               r.conname,
-               r.contype,
-               unnest(r.conkey) AS conkey
```

Comment:
Seems this was a leftover from a while ago - the PKs/FKs are now obtained in other queries.